src/ios/BlinkIDReactNative/BlinkIDReactNative/BlinkIDReactNative.mm

```diff
@@ -52,6 +52,7 @@ @interface BlinkIDReactNative () <PPScanningDelegate>
 static NSString* const kOptionReturnCroppedImageJsKey = @"shouldReturnCroppedImage";
 static NSString* const kOptionShouldReturnSuccessfulImageJsKey = @"shouldReturnSuccessfulImage";
 static NSString* const kOptionReturnFaceImageJsKey = @"shouldReturnFaceImage";
+static NSString* const kOptionQuality = @"quality";
```

Comment:
@mvayngrib 
Is this needed? I don't see it being used.

Reply:
right, i think i was considering making "quality" a parameter and forgot. I'll delete